### ISSUE TITLE
React 17 support

### ISSUE THREAD

**Issue by AndreiBehel** [FortAwesome/react-fontawesome#394]
Dependency conflict happen when a new react application is created

```
# npm resolution error report

2020-10-26T12:09:20.893Z

While resolving: project
Found: react@17.0.1
node_modules/react
  dev react@"^17.0.1" from the root project

Could not resolve dependency:
peer react@"16.x" from @fortawesome/react-fontawesome@0.1.11
node_modules/@fortawesome/react-fontawesome
  @fortawesome/react-fontawesome@"^0.1.11" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.
```

- react: 17.0.1
- react-fontawesome: 0.1.11

**Comment by robmadole** [comment]
I just released 0.1.12. Can you give this a try @AndreiBehel

**Comment by AndreiBehel** [comment]
> I just released 0.1.12. Can you give this a try @AndreiBehel

I've just tested it and it works good. Thanks!
